ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6486,4 +6486,18 @@ RFVSC2X:
   - return_fan_speed_percentage_command_1
   - return_fan_speed_percentage_command_2
   implements:
-  - OPERATIONAL
\ No newline at end of file
+  - OPERATIONAL
+
+DSPZTC:
```

Comment:
Is there a reason you are using this over [DSP](https://github.com/google/digitalbuildings/blob/afcd129422941b8c4d50bcf8e268b251b1c700fb/ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml#L256)?

high and low limit setpoints for zone air are basically what the heating and cooling setpoints on DSP do. Plus, some of the other stuff related to the discharge fan need to come out of this anyway since this functional control is really around zone temp control, not fan control.  In any case, the failed discharge fan and run status need to come out of this type. I would also recommend making the temperature alarms optional.